src/server.c

```diff
@@ -279,6 +281,18 @@ int dictSdsKeyCompare(dict *d, const void *key1,
     return memcmp(key1, key2, l1) == 0;
 }
 
+int dictSdsMstrKeyCompare(dict *d, const void *sdsLookup, const void *mstrStored)
+{
+    int l1,l2;
+    UNUSED(d);
+
+    l1 = sdslen((sds)sdsLookup);
+    l2 = hfieldlen((hfield)mstrStored);
```

Comment:
I'm not a fan of this implement.
Mixing sds and mstr feels confusing, and I prefer to use mstr exclusively rather than mixing them.

Reply:
This is the place where both got to meet. Here, the `sds` is the lookup value from "outside" into the hash which stores `mstr`.